release-notes/9.0/os-packages.json

```diff
@@ -0,0 +1,458 @@
+{
+    "$schema": "../schemas/dotnet-os-packages.json",
+    "channel-version": "9.0",
+    "eol-date": "2026-05-11",
```

Comment:
What's the need for this? This seems to be overloading the data represented here with data whose source of truth is in a separate file. It seems irrelevant to package descriptions.

Reply:
I wondered about that, too. I'll remove it.